services/gundeck/src/Gundeck/API/Error.hs

```diff
@@ -26,6 +26,3 @@ notificationNotFound = mkError status404 "not-found" "Some notifications not fou
 
 clientError :: Text -> Error
 clientError = mkError status400 "client-error"
-
```

Comment:
from what i see this entire module can be removed.

Reply:
good catch